index.html

```diff
@@ -97,33 +97,52 @@
 <body>
   <section id="abstract">
     <p>
-      The present document defines a WoT Profile that enables out of the box interoperability
-      among things and devices. It defines a set of constraints and rules that compliant
-      thing descriptions have to adopt to ensure interoperability.
+      The <a>WoT Profile Specification</a> defines a <em>WoT Core Profile</em>, 
+      which enables <em>out of the box interoperability</em>
+      among things and devices. <em>Out of the box interoperability</em> implies,
+      that devices can be integrated into various application scenarios without 
+      deep level adaptations. Typically only minor configuration operations are 
+      necessary (such as entering a network key, or IP address) 
+      to use the device in a certain scenario. 
+      These actions can be done by anyone without specific training.
+    </p>
+      The <a>WoT Core Profile</a> defines a set of <em>constraints 
+      and rules</em>, which compliant thing descriptions have to adopt 
+      to guarantee interoperability.
     </p>
     <p>
       These rules are prescriptive, to ensure that
-      compliant implementations satisfy the semantic guarantees,
-      that are implied by them. We call this set of rules a <em>Profile</em>.
+      compliant implementations satisfy the semantic guarantees
+      implied by them. We call this set of rules a <em>Profile</em>.
     </p>
     <p>
       The <strong>WoT Profile Specification</strong> as defined in
-      the present document serves two purposes:
+      this document serves two purposes:
     </p>
     <ul>
       <li>It defines a generic <strong>Profiling
           Mechanism</strong> which provides a mechanism to describe a
         profile in an unambiguous way. This mechanism can be
         used to define additional profiles.
       </li>
-      <li>In addition it defines a <strong><a>Core
-            Profile</a></strong> of the Thing Description for use with
-        the HTTP(S) and selected notification protocols. The <a>Core
+      <li>In addition, it defines a <strong><a>WoT Core
+            Profile</a></strong> of the Thing Description, which
+        consists of a core data model and protocol
+        binding rules. The <a>WoT Core
           Profile</a> formalizes the results of several PlugFests
         that were conducted by the WoT Interest Group and of
-        tests that were conducted as part of the development. It
-        is expected that additional profiles for thing templates
-        and other protocols will be defined in the near future.
+        tests that were conducted as part of the development.
+        <p>
+          This document incudes a binding of the core data model to
+          HTTP(S) and selected notification protocols.
```

Comment:
again, this is very confusing. it sounds if there are other protocols used then HTTP. The intention is to define event based mechanism based on HTTP. Formally correct we call this in the WoT context "sub-protocols" such as long poll etc.